QuackNBOOM/QuackNBOOM/DuckController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  DuckController.swift
+//  QuackNBOOM
+//
+//  Created by Ngo Tuyetnhi N. on 3/8/18.
+//  Copyright © 2018 Neriah and Neenee. All rights reserved.
+//
+
+import Foundation
+import SpriteKit
+
+class DuckController{
+    //just have one duck for now then have about 3-5 later
+    let duck = Duck(imagePath: "Duck")
+    
+    required init(){
+        //set the initial position of the duck above the screen
+        duck.pos = CGPoint(x: 1000, y: 1750) //temp values, change later
+    }
+    
+    //returns all ducks on current controller
+    public func getAllDucks()->Duck{
```

Comment:
Public is for things that are needing scope outside of the namespace, essentially this method is already in scope of your entire project without the use of the public keyword.  That being said if you wish to hide the other methods from the rest of the project use the private or fileprivate keyword.

Reply:
**The requested change has been successfully implemented**
The functions within the scope has been properly changed to fileprivate